client/tests/jest/jest.setup.js

```diff
@@ -32,4 +72,29 @@ global.setImmediate = global.setTimeout;
 jest.mock("@/composables/hashedUserId");
 jest.mock("@/composables/userLocalStorage");
 
+// Filter out specific vue2-teleport errors before failOnConsole processes them
+// These are harmless cleanup errors that occur after tests complete
+const originalConsoleError = console.error;
```

Comment:
We have a bunch of helpers for specific error suppression on a per test case that I think are more targeted in helpers.js - I would prefer going that route for consistency and more isolation but this is fine. I feel like I have seen this error though and I feel like I worked around it by just cleaning everything up from the test - I feel like when I saw this is was an indication things were running after the test was complete. Maybe I'm wrong though.